README.md

```diff
@@ -39,7 +39,13 @@ You can view the installation log by running
 
 ### A note on initial setup
 
-The API username and API password you are prompted for are credentials for your Sonar instance. You should create a dedicated user to utilize for the customer portal - **do not use your admin username/password!** The user requires account create, read, update, and delete permissions, ticket create, read, and update permissions, ticket super user permissions, and financial read permissions.
+The API username and API password you are prompted for are credentials for your Sonar instance. You should create a dedicated user to utilize for the customer portal - **do not use your admin username/password!** 
+
+**This permissions are different then v1 portal permissions**
```

Comment:
`This` should be `These`. `Then` should be `than`.

Reply:
Corrected